data/json/monsters/zed_lieutenant.json

```diff
@@ -114,8 +114,8 @@
     "max_accuracy": 20,
     "min_range": 30,
     "max_range": 30,
-    "//": "14 moves worth of casting time. It tends to stick 10 tiles away from you, so this means you can get some attacks in if you chase it. Currently toggled off, because it's too easy to run this thing down if it pauses while you move right at it.",
-    "//base_casting_time": "2800",
+    "//": "A little bit of cast 'backswing' so that it can be caught in more situations.",
+    "base_casting_time": "800",
```

Comment:
fyi monsters do not use `base_casting_time`, they make their attack instantly, and are ruled solely by `cooldown` field

Reply:
Really? That's annoying, but I might as well leave it in.

Reply:
I don't see the reason, it will do nothing but confuse people
```suggestion
```